core-scala/src/test/scala/com/baeldung/scala/LazyValTest.scala

```diff
@@ -0,0 +1,43 @@
+package com.baeldung.scala
+
+import org.scalatest.FunSuite
+import org.scalatest.Matchers._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+import scala.concurrent.{Future, _}
+import com.baeldung.scala
+
+class LazyValTest extends FunSuite {
```

Comment:
The file (and class name) should end in UnitTest

Reply:
The name of the file has been changed.